tests/_utils.py

```diff
@@ -27,6 +29,9 @@
 else:
     import pandas as pd
 
+    if version.parse(pd.__version__) >= version.parse("2.0.0"):
+        is_pandas_2_x = True
+
```

Comment:
Do you think it's worth adding a warning here along the lines of: "We have detected that your are using pandas 2.x, please note that we don't..."